stack.yaml

```diff
@@ -1,11 +1,10 @@
-resolver: lts-13.14
+resolver: lts-16.15
 
 packages:
   - '.'
 
 # Dependency packages to be pulled from upstream that are not in the resolver
-extra-deps:
-- scotty-cookie-0.1.0.3@sha256:3ff1df13a5acba8ba170a5ac22b3ac6a2c227791292536ce1ebfc41038f58fc9
+extra-deps: []
```

Comment:
`scotty-cookie` is needed because `stack build --flag oidc-client:build-examples` is going to fail.

Could you please see ["Run example"](https://github.com/krdlab/haskell-oidc-client#run-example)?